package.json

```diff
@@ -11,7 +11,9 @@
     "test": "npm run test:lint && npm run test:unit && npm run test:smoke",
     "test:lint": "standard",
     "test:unit": "tape tests/**/*.test.js | faucet",
-    "test:smoke": "tape tests/*.smoketest.js | faucet"
+    "test:smoke": "tape tests/*.smoketest.js | faucet",
+    "snyk-protect": "snyk protect",
+    "prepublish": "npm run snyk-protect"
```

Comment:
@TheAlphaNerd not always, but most times: 
<img width="676" alt="screen shot 2016-07-09 at 23 32 30" src="https://cloud.githubusercontent.com/assets/6078550/16710415/76647432-462d-11e6-8861-3ec541394986.png">